composer.lock

```diff
@@ -17949,6 +17443,5 @@
     },
     "platform-overrides": {
         "php": "7.1.9"
-    },
-    "plugin-api-version": "1.1.0"
+    }
```

Comment:
This seems to have been generated with an older version of Composer. Please regenerate this using the version of Composer that is bundled with Joinup.